src/templates/queues/form.html

```diff
@@ -0,0 +1,30 @@
+{% extends "queues/base_queues.html" %}
+
+{% block head_title %}Queue form{% endblock %}
+{% block page_title %}<h2>Queue form</h2>{% endblock page_title %}
+
+{% block content %}
+    <div class="ui container message">
+        <form class="ui form" action="" method="post">
+            {% csrf_token %}
+            <table class="ui table table-bordered">
+                {{ form.as_table }}
```

Comment:
why are we doing this with forms? Shouldn't we be doing this like everything else in v2? with the API?